spring-cloud-dataflow-docs/src/main/asciidoc/applications.adoc

```diff
@@ -3,126 +3,4 @@
 
 [partintro]
 --
-A selection of pre-built link:https://cloud.spring.io/spring-cloud-stream-app-starters/[stream] and link:https://cloud.spring.io/spring-cloud-task-app-starters/[task or batch] starter applications for various data integration and processing scenarios to facilitate learning and experimentation. The table in the next section includes the pre-built applications at a glance. For more details, review how to <<index.adoc#supported-apps-and-tasks, register supported applications>>.
```

Comment:
This removal is really part of https://github.com/spring-cloud/spring-cloud-dataflow/issues/4807